### DISCUSSION TITLE
Application error: a client-side exception has occurred (see the browser console for more information).

### DISCUSSION THREAD

**Discussion by PittaVision** [WalletConnect/walletconnect-monorepo#5145] [Developer Support]
I'm using Wallet Connect as my Web3 wallet RPC provider for staking chainlink and I keep getting the "Application error: a client-side exception has occurred (see the browser console for more information)." message

I also get "We detected a problem with your web3 wallet RPC provider. The application may not behave as expected." before the webpage goes to the "Application error: a client-side exception has occurred (see the browser console for more information)." message

What is the issue?

**Comment by glitch-txs** [comment]
please could you provide a minimal reproducible example of the issue?
